components/shapediver/parameter/ParameterColorComponent.tsx

```diff
@@ -71,16 +72,24 @@ export default function ParameterColorComponent(
 		(state) => state.uiValue,
 	);
 
+	const {onFocusHandler, onBlurHandler, restoreFocus, focusedElement} =
+		useFocus();
+
 	const handleSdColorChange = useCallback(
 		(val: string) => {
-			handleChange(convertToSdColor(val, colorFormat));
+			handleChange(
+				convertToSdColor(val, colorFormat),
+				undefined,
+				restoreFocus,
+			);
 		},
-		[handleChange, colorFormat],
+		[handleChange, colorFormat, focusedElement],
```

Comment:
@LemToUp I think here it should be `restoreFocus` instead of `focusedElement`

Reply:
Done.